packages/expo-video/ios/VideoView.swift

```diff
@@ -12,21 +12,28 @@ public final class VideoView: ExpoView, AVPlayerViewControllerDelegate {
     }
   }
 
+  var wasPlaying: Bool = false
```

Comment:
Might be a good idea to put this property inside `#if os(tvOS)` as well, since it's used only on this platform. Its value may not be correct on other platforms, so it could be confusing.

Reply:
Done!